src/components/special/_toggle.less

```diff
@@ -0,0 +1,38 @@
+.toggle-btn {
+  width: 90px;
+  height: 50px;
+  border-radius: 50px;
+  border: 0;
+  transition: all 0.5s ease;
+  outline: none;
+  display: flex;
```

Comment:
Change display to `inline-flex`

Reply:
@shahednasser should I close this pull request and create a new one?

Reply:
Sure if there are too many conflicts then do that

Reply:
okay! Ill keep in mind these changes that u have suggested and create a new pr